Session/Shared/Types/SessionProState.swift

```diff
@@ -0,0 +1,17 @@
+// Copyright © 2025 Rangeproof Pty Ltd. All rights reserved.
+
+import SessionUIKit
+import SessionUtilitiesKit
+
+public class SessionProState: SessionProCTADelegate {
```

Comment:
Would the idea be that this is replaced by a dependency that conforms to `SessionProCTADelegate`? (eg. you'd pass `delegate: dependencies[singleton: .proState]` once we have some proper logic)